force-app/main/default/labels/CustomLabels.labels-meta.xml

```diff
@@ -13596,14 +13592,6 @@ Selecting Don&apos;t Auto Schedule Default NPSP Jobs tells NPSP not to check for
         <shortDescription>stgHelpAdvancedMapping2</shortDescription>
         <value>You can disable Advanced Mapping and go back to Help Text mapping, but any changes you made with Advanced Mapping will be lost.</value>
     </labels>
-    <labels>
-        <fullName>stgHelpAdvancedMapping3</fullName>
-        <categories>Settings</categories>
```

Comment:
These were a bit funky to deal with in removing them, I'm not sure if we want to remove the entire label or just the value, can definitely just remove the value of course!

Reply:
There won't be any problem with removing a custom label from the package as long as all references and translations are removed. The only catch is that the labels aren't actually removed from the existing customers' orgs, it's will only be removed in the brand new fresh org.

Reply:
Okay, thank you for clarifying that @andrewyu-salesforce , going to make sure I remove all label references for these!